pages/index.js

```diff
@@ -53,6 +54,8 @@ export default function Home() {
         ) : (
           posts.map((post, i) => {
             post = post.attributes;
+            var [date, _] = formateDate(post.publishedAt);
```

Comment:
We are doing the same thing when we get an API response. Let's use that field only. You can replace **published_on** to **publishedAt** in `setPostFields`